src/sdk/pynni/nni/bohb_advisor/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/QuanluZhang/ConfigSpace.git
```

Comment:
why not we pull the original ConfigSpace?

Reply:
In order to be consistent with SMAC ...